include/caf/events/force_power_down_event.h

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Wondering if it won't be better to keep power manager related events in a single file. Also we could think about some common prefix for them.

Anyway - this can be done later as cleanup while documentation is added,

Reply:
In my feeling it is better to split it as it makes it easier to trace the code and capsule it.